qa/evil-tests/src/test/java/org/opensearch/action/main/EvilSystemPropertyTests.java

```diff
@@ -0,0 +1,36 @@
+/*
```

Comment:
The name if this file and class is 💀 🔥

Reply:
It is intentional, but I can't claim it as my own suggestion :)  There are existing similar tests under [/cluster/metadata](https://github.com/mch2/OpenSearch/blob/main/qa/evil-tests/src/test/java/org/opensearch/cluster/metadata/EvilSystemPropertyTests.java) and [cluster/routing](https://github.com/mch2/OpenSearch/blob/main/qa/evil-tests/src/test/java/org/opensearch/cluster/routing/EvilSystemPropertyTests.java)

Reply:
Oh wow, it's a thing!